practice_app/pages/api/getStatisticsModule.js

```diff
@@ -0,0 +1,16 @@
+exports.checking = function (val) {
+    let cnull = 'no'
+    let ansnull = 'yes'
+
+    if (val.countryN == null) {
+        cnull = 'yes'
+        return { cnull, ansnull }
+    }
+
+    if (val.answer == null) {
+        ansnull = 'no'
+        return { cnull, ansnull }
+    }
+
+    return { cnull, ansnull }
+}
```

Comment:
This function is unnecessarily long, Furkan. This can be a one-liner: 
```js
return { cnull: val.countryN == null, ansnull: val.ansnull == null}
```

I would say get rid of this function and make a function that does more than this.

Reply:
I got rid of it and created a better function. Here is unit tests of this new function:
> template_next@0.1.0 test C:\Users\furka\Documents\GitHub\bounswe2020group2\practice_app
> jest --passWithNoTests

 PASS  services/getLocationInfo.spec.js
 PASS  pages/api/getStatistics.test.js

Test Suites: 2 passed, 2 total
Tests:       12 passed, 12 total
Snapshots:   0 total
Time:        1.953 s
Ran all test suites.

C:\Users\furka\Documents\GitHub\bounswe2020group2\practice_app>